src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,42 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.Arrays;
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int REQUIRED_AGE = 35;
+    private static final String REQUIRED_NATIONALITY = "Ukrainian";
+    private static final int REQUIRED_YEARS_IN_UKR = 10;
+
+    @Override
+    public boolean test(Candidate c) {
```

Comment:
never use such naming - just a letter or digit

```suggestion
    public boolean test(Candidate candidate) {
```